eng/pipelines/docindex.yml

```diff
@@ -2,3 +2,44 @@ trigger: none
 
 jobs:
   - template: /eng/common/pipelines/templates/jobs/docindex.yml
+
+  - job: UpdateDocsMsBuildConfig
+    pool:
+      vmImage: ubuntu-20.04
```

Comment:
Do you not want to use the 1es pool?

Reply:
We could. I have no strong opinions. 

Arguments for using the hosted pool for engsys tools as opposed to the 1ES pools: 
* 1ES pools may saturate from time to time and using a different pool gives us the ability to run some processes without having to wait for machines to become available 
* The capacity in these pools can be used at comparatively lower cost